app/src/main/java/com/sdp13epfl2021/projmag/model/ImmutableProject.kt

```diff
@@ -29,11 +27,13 @@ data class ImmutableProject constructor(
     val tags: List<String>,
     val isTaken: Boolean,
     val description: String,
-    val videoURI: List<String> = listOf()
+    val videoURI: List<String> = listOf(),
+    val allowedSections: List<String> = listOf()
 ) : Parcelable {
     companion object {
-
-        object FieldNames {
+        private val sectionsManager = SectionBaseManager()
+        private val tagsManager = TagsBaseManager()
```

Comment:
Is there a reason to keep these managers here ?
They are private and never used.